### PR TITLE
Exercise 1: Opioids Exercise

### PR DESCRIPTION
**Pull request by radamson**
WIP Opioids Notebook

See the notebook here: https://github.com/mitre/fhir-exercises/blob/opioids-1/Opioids%20Use%20Case%20Example.ipynb

### CODE REVIEW

**Review by jawalonoski — COMMENTED** [review]
- Add Learning Objectives and Key Concepts before Introduction.
- "reasonable return" should be "reasonably return"
- Remove or resolve TODO: regarding dynamic medication name

Overall, looks good.

**Comment by radamson** [comment]
> - [x] Add Learning Objectives and Key Concepts before Introduction.
> - [x] "reasonable return" should be "reasonably return"
> - [x] Remove or resolve TODO: regarding dynamic medication name

Addressed the comments. Instead of making the medication name dynamic, I added a list comprehension to check for a tramadol MedicationRequest since I think making it dynamic might make the rest of the narrative harder, especially if the first medication returned ends up not being an opioid

**Comment by radamson** [comment]
Made a few small additions. The main things:

Added an `Icons in this Guide` section:
Screen Shot 2022-01-27 at 3 13 02 PM

Added a very basic FHIR Terminology API call (`$lookup`) and links to further documentation.
Screen Shot 2022-01-27 at 3 14 25 PM

**Comment by radamson** [comment]
Added `DataFrame.describe` to align with `skim` used in the R notebook

Screen Shot 2022-01-27 at 3 58 15 PM

**Comment by radamson** [comment]
https://github.com/mitre/fhir-exercises/pull/2/commits/6650f26bc71d272fa55efa3f0f2e8462a0eec931  - Replaced Pie Charts with Tables to better align with R notebook

Screen Shot 2022-01-27 at 5 19 13 PM
.

**Comment by radamson** [comment]
https://github.com/mitre/fhir-exercises/pull/2/commits/23bef06914464d69c39c912a0817dca773e2a03a - More updates based on the R notebook.

Defined `get_atc_class` method and applied it to all rows.

Screen Shot 2022-01-27 at 5 52 15 PM

**Review by jawalonoski — APPROVED** [review]
The commit I just made fixes some minor typos in the narrative, and adds a short recap at the end that essentially repeats the learning objectives to close out the notebook.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,45 @@
...
+pip install -r requirements.txt
+```
+
+**Run JupyterLab**
+
+```shell
+jupyter-lab
+```
+
+By default, this should open Jupyter lab at
```
Comment (dehall) [inline README.md]:
```suggestion
By default, this should open Jupyter lab at http://localhost:8888/lab
```

**Pull request merged**